package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "game-places",
+  "name": "offer-up",
   "version": "0.1.0",
```

Comment:
Should we manually bump the version? Your PR title denotes a breaking change but I'm not sure CI is set up to care about that.

Reply:
The breaking change indicator is because the contract API surface changed in a way that broke the ui (along with any other clients that may exist).

As to versions... I try not to learn any more npm version stuff than I have to. I have not been forced to learn enough to indicate that the version should change here. The machines have not complained.